app/services/subscriptions/dates/yearly_service.rb

```diff
@@ -100,6 +100,13 @@ def compute_charges_duration(from_date:)
 
         compute_duration(from_date:)
       end
+
+      def period_started_in_last_year?(date)
+        return true if (date.month < subscription_at.month)
```

Comment:
Style/ParenthesesAroundCondition: Don't use parentheses around the condition of an `if`. (https://rubystyle.guide#no-parens-around-condition)